src/plugins/home/server/plugin.ts

```diff
@@ -19,15 +19,18 @@ import { UsageCollectionSetup } from '../../usage_collection/server';
 import { capabilitiesProvider } from './capabilities_provider';
 import { sampleDataTelemetry } from './saved_objects';
 import { registerRoutes } from './routes';
+import { CustomIntegrationsPluginSetup } from '../../custom_integrations/server';
 
-interface HomeServerPluginSetupDependencies {
+export interface HomeServerPluginSetupDependencies {
   usageCollection?: UsageCollectionSetup;
+  customIntegrations: CustomIntegrationsPluginSetup;
```

Comment:
nit: declare required properties before optional ones

Reply:
changed to optional dependency